docs/CONFIGURATION.md

```diff
@@ -16,4 +16,51 @@ The default configuration is:
 thematics: './thematic/*.thematic.mdx'
 datasets: './datasets/*.data.mdx'
 discoveries: './discoveries/*.discoveries.mdx'
+```
+
+### Component overrides
```

Comment:
I think this title should be page overrides? and make it clear that there are 2 kinds (component override, content override) of overrides for page overrides. 
It might be even better to add a clear note that both of these overrides can be defined in the `delta.config.js` under `pageOverrides`.

Reply:
Yes, the title is wrong. Will fix.

The types of overrides are being listed as bullet items, and then each one is explained in detail. Currently there is only the `aboutContent` but we'll update the docs as we add more. Would you organize this differently?

Reply:
No it was more about wording. maybe an example pageOverrides config including componentOverride can be more helpful than the current one? (but feels like all these are nitpicks.)